lib/kubernetes-deploy/kubectl.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'kubeclient'
+
+module KubernetesDeploy
+  module Kubectl
+    class KubectlError < FatalDeploymentError; end
+
+    def run_kubectl(*args, namespace: "", context: "", namespaced: true, with_context: true)
```

Comment:
It was kept to keep comaptibility with the usage in the `Runner` class since I was really just testing this out. However, since I override it anyways, I'll change this.